types.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
   error: {
     badge: figures.cross,
     color: 'red',
-    label: 'error'
+    label: 'error',
+    stream: process.stderr
```

Comment:
This should be omitted, in order to prevent confusion, since default logger are all expected to write to the default `process.stdout` stream